### PR TITLE
created order model

### PR DESCRIPTION
**Pull request by HubDub**
## Pull Request Description

Created order model. 
## Steps to Test

git checkout dbOrderModel
examine model code for any errors
#5

### CODE REVIEW

**Review by Jgregnier — COMMENTED** [review]
The Order model is named Customer on line 8

File: Models/OrderModel.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Bangazon.Models
+{
+    public class Order
+    {
+        [key]
```
Comment (delaineackerman) [inline Models/OrderModel.cs]: I believe this needs to be Uppercase - like [Key]

File: Models/OrderModel.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        public int OrderId { get; set}
+
+        [Required]
+        public bool IsCompleted {get; set}
+        public int? PaymentTypeId {get; set}
+        public PaymentType PaymentType {get; set}
+
+        [Required]
```
Comment (delaineackerman) [inline Models/OrderModel.cs:18]: This does not need to be Required

File: Models/OrderModel.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        public int CustomerId {get; set}
+        public Customer Customer {get; set}
+        public ICollection<LineItem> LineItems; 
+
+        [Required]
+        [DataTypeAttribute(DataType.Date)]
```
Comment (delaineackerman) [inline Models/OrderModel.cs:24]: Should be DataType rather than DataTypeAttribute

File: Models/OrderModel.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        [DatabaseGeneratedAttribute(DatabaseGeneratedOption.Computed)]
```
Comment (delaineackerman) [inline Models/OrderModel.cs:25]: Should be DatabaseGenerated rather than DatabaseGeneratedAttribute

File: Models/OrderModel.cs [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        public DateTime DateCreated {get;set;}
+
+        [Required]
+        [DataTypeAttribute(DataType.Date)]
```
Comment (delaineackerman) [inline Models/OrderModel.cs:29]: See comments above for DataType and DatabaseGenerated

**Review by Jgregnier — APPROVED** [review]
Changed key to Key now. Should be good to merge

File: Models/OrderModel.cs (hunk 2 above)
Comment (delaineackerman) [reply Models/OrderModel.cs:18]: Should be required (disregard)

File: Models/OrderModel.cs (hunk 4 above)
Comment (delaineackerman) [reply Models/OrderModel.cs:25]: disregard

File: Models/OrderModel.cs (hunk 5 above)
Comment (delaineackerman) [reply Models/OrderModel.cs:29]: disregard

**Pull request merged**